www/app.py

```diff
@@ -74,6 +128,7 @@ def run_command(decoded):
         print "Stopping motor"
         pwm.set_duty_cycle(decoded['pin'], 0)
 
+init_rover_service()
```

Comment:
Yeah. When app gets imported, it gets run.  I had to take it out of the `if __name__ == '__mail'`, because that doesn't get called when uwsgi runs app.